delta/delta.go

```diff
@@ -0,0 +1,25 @@
+package delta
+
+import (
+	"github.com/tufin/oasdiff/diff"
+)
+
+const coefficient = 0.5
+
+func Get(asymmetric bool, diffReport *diff.Diff) float64 {
+	if diffReport.Empty() {
+		return 0
+	}
+
+	delta := getEndpointsDelta(asymmetric, diffReport.EndpointsDiff)
+
+	return delta
+}
+
+func ratio(asymmetric bool, added int, deleted int, modifiedDelta float64, all int) float64 {
+	if asymmetric {
+		added = 0
+	}
+
+	return (float64(added+deleted) + modifiedDelta) / float64(all)
+}
```

Comment:
Maybe can add a safety check in case all == 0:
```suggestion
func ratio(asymmetric bool, added int, deleted int, modifiedDelta float64, all int) float64 {
	if asymmetric {
		added = 0
	}
	if all == 0 {
		all = 1
	}

	return (float64(added+deleted) + modifiedDelta) / float64(all)
}
```

Reply:
Thanks for the feedback.
In practice `all` can only be zero if both compared collections are empty and that should be caught by the `if d.Empty()` check at the beginning of each getXXXDelta function.